pkg/cluster/cluster.go

```diff
@@ -93,6 +94,7 @@ func (c *clusterOpts) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.baseDir, "base-dir", c.baseDir, "path of the base dir to store cluster data")
 	fs.StringVarP(&c.sshKeyPath, "ssh-path", "s", c.sshKeyPath, "path to public ssh key for cluster")
 	//fs.StringVar(&c.keepBootstrap, "keep-bootstrap", r.keepBootstrap, "keep boostrap node around for debug")
+	fs.BoolVarP(&c.daemonMode, "daemon", "d", false, "In daemon mode, output is redirected to a log file")
```

Comment:
I think we should keep noncritial params as long-only `fs.BoolVar `and save short for things we would use often.